packages/esm-ward-app/src/ward-workspace/admission-requests.workspace.tsx

```diff
@@ -1,37 +1,55 @@
 import React from 'react';
 import styles from './admission-requests-workspace.scss';
-import AdmissionRequestCard from './admission-request-card.component';
-import { getCoreTranslation, type DefaultWorkspaceProps } from '@openmrs/esm-framework';
+import AdmissionRequestCard from './components/admission-request-card.component';
+import { type InpatientRequest } from '../types';
+import { Search } from '@carbon/react';
+import { ErrorState, isDesktop, useLayoutType } from '@openmrs/esm-framework';
+import { Layout } from '@carbon/react/lib/components/Layout';
+import { useTranslation } from 'react-i18next';
 import { useInpatientRequest } from '../hooks/useInpatientRequest';
-import useWardLocation from '../hooks/useWardLocation';
-import { Loading } from '@carbon/react';
 import { InlineNotification } from '@carbon/react';
-import { useTranslation } from 'react-i18next';
+import useWardLocation from '../hooks/useWardLocation';
 
-const AdmissionRequestsWorkspace: React.FC<DefaultWorkspaceProps> = () => {
+interface AdmissionRequestsWorkspaceProps {}
+const AdmissionRequestsWorkspace: React.FC<AdmissionRequestsWorkspaceProps> = () => {
+  const {
+    inpatientRequests,
+    isLoading: isLoadingInpatientRequests,
+    error: errorFetchingInpatientRequests,
+  } = useInpatientRequest();
   const { t } = useTranslation();
-  const { inpatientRequests, isLoading, error } = useInpatientRequest();
-  const admissionRequests = inpatientRequests?.filter((request) => request.dispositionType == 'ADMIT');
-
-  if (isLoading) {
-    return <Loading />;
-  }
-
-  if (error) {
-    return (
-      <InlineNotification
-        kind="error"
-        title={t('errorLoadingPatientAdmissionRequests', 'Error Loading patient admission requests')}>
-        {getCoreTranslation('errorCopy')}
-      </InlineNotification>
-    );
-  }
+  const [searchTerm, setSearchTerm] = React.useState('');
+  const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchTerm(event.target.value);
+  };
 
   return (
-    <div className={styles.admissionRequestsWorkspace}>
-      {admissionRequests.map((admissionRequest, indx) => (
-        <AdmissionRequestCard key={indx} patient={admissionRequest.patient} />
-      ))}
+    <div className={styles.admissionRequestsWorkspaceContainer}>
+      <Search
+        labelText=""
+        value={searchTerm}
+        onChange={handleSearch}
+        size="lg"
+        placeholder={t('searchForPatient', 'Search for a patient')}
+        disabled
+      />
```

Comment:
Is this search field doing anything yet?  Looks like it isn't, but I wanted to confirm.

Reply:
No @mogoodrich , it isn't doing anything, but I have kept it there and disabled it for the time being.

Reply:
Tangentially, after this ticket is merged we should discuss with Fiona and others what we want this search to do before implementing, as I don't think it's been fully designed out. (But not to worry for this ticket).